shell/browser/api/electron_api_session.cc

```diff
@@ -780,6 +780,20 @@ bool Session::AddWordToSpellCheckerDictionary(const std::string& word) {
 #endif
   return service->GetCustomDictionary()->AddWord(word);
 }
+
+bool Session::RemoveWordFromSpellCheckerDictionary(const std::string& word) {
+#if BUILDFLAG(USE_BROWSER_SPELLCHECKER)
+  if (spellcheck::UseBrowserSpellChecker()) {
+    spellcheck_platform::RemoveWord(base::UTF8ToUTF16(word));
```

Comment:
Build is failing at this line:

```
../../electron/shell/browser/api/electron_api_session.cc(787,60): error: too few arguments to function call, expected 2, have 1
33730    spellcheck_platform::RemoveWord(base::UTF8ToUTF16(word));
33731    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                        ^
33732../..\components/spellcheck/browser/spellcheck_platform.h(111,1): note: 'RemoveWord' declared here
33733void RemoveWord(PlatformSpellChecker* spell_checker_instance,
33734^
337351 error generated.
33736
```

Reply:
@erickzhao the rebase on master pulled in a Chrome upgrade that changed the signature here, check out the modifications to `AddWord` to see what changes need to be made